mock-certify-plugin/src/main/java/io.mosip.certify.mock.integration/service/MockVCIssuancePlugin.java

```diff
@@ -235,10 +240,48 @@ private static String getUTCDateTime() {
 	@Override
 	public VCResult<String> getVerifiableCredential(VCRequestDto vcRequestDto, String holderId,
 													Map<String, Object> identityDetails) throws VCIExchangeException {
+		String accessTokenHash = identityDetails.get(ACCESS_TOKEN_HASH).toString();
+        Map<String, Object> data = new HashMap<>();
+        String documentNumber;
+        try {
+			documentNumber = getIndividualId(getUserInfoTransaction(accessTokenHash));
+		} catch (Exception e) {
+			log.error("Error getting documentNumber", e);
+			throw new VCIExchangeException(ErrorConstants.VCI_EXCHANGE_FAILED);
+		}
+
+        log.info("Setting up the data for mDoc");
```

Comment:
Extract the static data to a separate function, to be replaced with db call in future

Reply:
Extraction is made